src/DevTools/FileReflection.php

```diff
@@ -176,41 +176,33 @@ public function get_file_source( $file ) {
 				$this->get_template_slug(),
 				$matches['file']
 			);
-		}
-
-		if ( $this->is_child_theme_file( $file, $matches ) ) {
+		} elseif ( $this->is_child_theme_file( $file, $matches ) ) {
```

Comment:
Why this change to `elseif`s? The individual conditions already end in `return`s...

Reply:
Reverted as discussed. As you shared: https://github.com/kalessil/phpinspectionsea/blob/master/docs/control-flow.md#redundant-else-keyword